src/care/CHAICallback.cpp

```diff
@@ -294,7 +296,9 @@ namespace care {
                   }
 
                   if (s_log_data > 0) {
+#ifndef CARE_DISABLE_RAJAPLUGIN
```

Comment:
Might as well have this outside the if statement. I think there are several places like this.